vignettes/advanced.Rmd

```diff
@@ -26,102 +29,290 @@ knitr::opts_chunk$set(
 The purpose of this vignette is to provide an overview of some of the more advanced
 features of the `rbmi` package. 
 
+# Data simulation {#sec:dataSimul}
+
+In order to demonstrate the advanced functions we will first create a simulated dataset with the `rbmi` function `simulate_data()`.
+A reader interested in other sections of the vignette must not necessarly read through this section.
+The `simulate_data()` function generates data for a two-arms clinical trial with longitudinal continuous outcome and two intercurrent events (ICEs).
```

Comment:
Maybe misleading as its "up to two ICEs per subject" its not guarenteed to generate 2 ICEs  (and ambigious if you mean overall or per patient

Reply:
Agreed